src/wtmp.c

```diff
@@ -40,7 +40,7 @@ int log_wtmp(const char *line, const char *name, const char *host,
     !(defined(LINUX) || defined(__hpux) || defined (_AIX))
   /* This "auxilliary" utmp doesn't exist under linux. */
 
-#if defined(__sparcv9) && !defined(__NetBSD__) && !defined(__FreeBSD__)
+#if defined(__sun)
```

Comment:
Hmm.  These platform-specific macros are from before I'd assumed maintainership of the project.  This issue makes me wonder whether now might be the time to move these into the `configure` script, so that this becomes more of a feature macro...